__tests__/src/reducers/windows.test.js

```diff
@@ -295,6 +295,20 @@ describe('windows reducer', () => {
     });
   });
 
+  describe('SET_ANNOTATIONS', () => {
+    it('replaces the annotations list for the given canvas by the given annotation', () => {
+      const beforeState = { abc123: { selectedAnnotations: { cId: ['prevId1', 'prevId2'] } } };
+      const action = {
+        type: ActionTypes.SET_ANNOTATIONS, windowId: 'abc123', canvasId: 'cId', annotationId: 'aId',
+      };
+      const expectedState = {
+        abc123: { selectedAnnotations: { cId: ['aId'] } },
+      };
+
+      expect(windowsReducer(beforeState, action)).toEqual(expectedState);
+    });
+  });
+
   describe('SELECT_ANNOTATION', () => {
```

Comment:
If we go this route, I'll update `SELECT_ANNOTATION` to be something like `ADD_SELECTED_ANNOTATION`.

Reply:
We did not go this route, so I've kept the action names the same.